src/goto-programs/goto_trace.cpp

```diff
@@ -149,6 +134,10 @@ std::string trace_value_binary(
       if(!to_integer(expr, i) && i>=0)
         return integer2string(i, 2);
     }
+    else if(type.id()==ID_string)
+    {
+      return integer2string(expr.get(ID_value).get_no(), 2);
```

Comment:
This doesn't work with `USE_STD_STRING` enabled: https://travis-ci.org/diffblue/cbmc/jobs/378446491#L1421